rasa/core/evaluation/marker.py

```diff
@@ -221,12 +216,12 @@ class SlotSetMarker(ConditionMarker):
     @staticmethod
     def positive_tag() -> Text:
         """Returns the tag to be used in a config file."""
-        return "slot_is_set"
+        return "slot_was_set"
 
     @staticmethod
     def negated_tag() -> Optional[Text]:
         """Returns the tag to be used in a config file for the negated version."""
-        return "slot_is_not_set"
+        return "slot_was_not_set"
```

Comment:
@ka-bu why did you decide to change them? After yesterday's discussion I felt that 'is' more accurately describes the behaviour (since slots must explicitly get unset for the condition to be false).

Reply:
because 
- it's used in rules and it is unnecessary extra work to remember to switch to those slightly diffferent terms
- that mismatch was also brought in the feedback excercise
- the "slot_is_set" was not the only thing -> "intent_detected" is also just "intent" in rules
- the meaning is *the exact same* as in rules

Reply:
👍